packages/legend-query-builder/src/components/__tests__/QueryBuilder_Milestoning.test.tsx

```diff
@@ -52,6 +52,7 @@ type TestCase = [
     rawLambda: { parameters?: object; body?: object };
```

Comment:
This is just a comment, but this kind of test unnerves me, it's strange because we're just checking the number of arguments provided. Anyhow, let's hope when we do #1149 we can reshape this test to something that is more solid 🤞